src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -68,12 +70,24 @@ public BinlogConnectorReplicator(
 			EventDeserializer.CompatibilityMode.CHAR_AND_BINARY_AS_BYTE_ARRAY);
 		this.client.setEventDeserializer(eventDeserializer);
 
-		this.binlogEventListener = new BinlogConnectorEventListener(client, queue);
+		Timer replicationQueueTimer = metrics.getRegistry().timer(metrics.metricName("replication", "queue", "time"));
+		this.binlogEventListener = new BinlogConnectorEventListener(client, queue, replicationQueueTimer);
+
 		this.client.setBlocking(!stopOnEOF);
 		this.client.registerEventListener(binlogEventListener);
 		this.client.setServerId(replicaServerID.intValue());
 
 		this.stopOnEOF = stopOnEOF;
+
+		metrics.getRegistry().register(
+				metrics.metricName("replication", "queue", "count"),
+				new Gauge<Long>() {
+					@Override
+					public Long getValue() {
+						return (long) queue.size();
```

Comment:
`queue` is blocked queue with capacity of 20, I'm feeling this metrics might not give us much value. Can we instrument the blocking time?